### DISCUSSION TITLE
Having trouble with turning my ship (maybe bug?)

### DISCUSSION THREAD

**Discussion by Flayute** [endless-sky/endless-sky#8634] [General]
Hey community, I've been playing this amazing game since 2019 and I'm sorry I didn't came in here before to congratulate you guys for your work. I've enjoyed hours of flight and fun. In every release I reset the pilot and start from the begining.

Well to the point. I've been playing to the new release for a few days now, but in the last days I lost the ability of turning my ship. When I set autopilot mode, land or board it turns with no problem, but then I try it with my keys and it doesn't work. 

I reinstalled the game twice, tried changing the key settings and it still there. 

I can't recall exactly when it started hapening, but I believe it started while doing the remnant misions.

When you have a huge and wild palavert fleet it's not a problem, but it is very annoying. :P

I believe it is a possible bug and I wanted to give my feedback for the first time ever. I hope it helps developing this amazing game. Thanks for making Escape Velovity real again and better

**Comment by Zireael07** [comment]
Try pressing alt. It's now a switch for mouse to turn and you probably accidentally switched it to on

**Reply by Flayute** [reply]
Wow, Thanks, so Easy. How do I turn it off? I can't find the option

**Reply by bene-dictator** [reply]
There's no way yet. The hotkey will be disabled in the next update because so many people keep pressing it by accident and filing bug reports : ) but do keep them coming

**Reply by Flayute** [reply]
Now I understand how it works, I'll give it a try.

Thanks for the fast answers

**Reply by Flayute** [reply]
> There's no way yet. The hotkey will be disabled in the next update because so many people keep pressing it by accident and filing bug reports : ) but do keep them coming

It is interesting, I changed the key settings for making it easier to use, but what it's going to be harder is changing my mind settings... :_)

**Comment by TomGoodIdea** [comment]
#8620
